helm_deploy/prison-offender-events/templates/_envs.tpl

```diff
@@ -88,76 +88,76 @@ env:
         name: dps-rds-instance-output
         key: rds_instance_endpoint
 
-  - name: SNS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_ACCESS_KEY_ID
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: access_key_id
 
-  - name: SNS_AWS_SECRET_ACCESS_KEY
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_SECRET_ACCESS_KEY
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: secret_access_key
 
-  - name: SNS_TOPIC_ARN
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_ARN
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: topic_arn
 
-  - name: HMPPS_SNS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_TOPICS_HMPPS_EVENT_ACCESS_KEY_ID
     valueFrom:
       secretKeyRef:
         name: hmpps-domain-events-topic
         key: access_key_id
 
-  - name: HMPPS_SNS_AWS_SECRET_ACCESS_KEY
+  - name: HMPPS_SQS_TOPICS_HMPPS_EVENT_SECRET_ACCESS_KEY
     valueFrom:
       secretKeyRef:
         name: hmpps-domain-events-topic
         key: secret_access_key
 
-  - name: HMPPS_SNS_TOPIC_ARN
+  - name: HMPPS_SQS_TOPICS_HMPPS_EVENT_TOPIC_ARN
     valueFrom:
       secretKeyRef:
         name: hmpps-domain-events-topic
         key: topic_arn
 
-  - name: SQS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_QUEUES_PRISON_EVENT_QUEUE_NAME
     valueFrom:
       secretKeyRef:
         name: prisoner-offender-events-queue
-        key: access_key_id
+        key: sqs_queue_name
 
-  - name: SQS_AWS_SECRET_ACCESS_KEY
+  - name: HMPPS_SQS_QUEUES_PRISON_EVENT_ACCESS_KEY_ID
```

Comment:
I think this one needs to be `HMPPS_SQS_QUEUES_PRISONEVENTQUEUE_QUEUE_ACCESS_KEY_ID` and similar for the others.

Reply:
yes - good spot - I can see that now